config/routes.rb

```diff
@@ -1,9 +1,14 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  root to: 'pages#index'
```

Comment:
**suggestion**: I generally prefer to have a controller for the home page. `pages#index` implies that we'll see a list of possible pages for `pages#show`. WDYT?

Reply:
Agreed! So i generated a new controller for the homepage and used show, going to keep pages for our miscellaneous pages.